### PR TITLE
Nayda n hw4

### PR DESCRIPTION
**Pull request by NathalieNayda**
work 4 is done

### CODE REVIEW

File: naydaN/HW4/HW4-1.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+year = int(input("input year"))
+if int(year % 4 == 0 and year % 100 == 0 and year % 400 == 0) :
```
Comment (katemalash) [inline naydaN/HW4/HW4-1.py:2]: What about 2020 year?

File: naydaN/HW4/HW4-2.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    weight = int(input ("weight = "))
+    height = int(input ("height = "))
+    print("square =",  weight * height )
+elif choice == "circle" :
+    import math
+    radius = int(input ("radius = "))
+    print('square =', math.pi * radius ** 2)
+elif choice == "triangle" :
+    S = int(input ("lenght = "))
+    a = int(input ("height = "))
```
Comment (katemalash) [inline naydaN/HW4/HW4-2.py:12]: It was required to input 3 sides of triangle

File: naydaN/HW4/HW4-3.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+number = str(input("Enter the number:"))
+digit = len(str(number))
+if int(number) > 0 :
+      print("number is positive", "lenght = ", digit)
+elif int(number) < 0 :
+      print("number is negative", "lenght = ", len(str(number))-1)
+else :
+      print("number is zero")
```
Comment (katemalash) [inline naydaN/HW4/HW4-3.py:8]: Nice Decision

File: naydaN/HW4/HW4-4.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+R = int(input("Radius " ))
+S = int(input("Square " ))
+K = int(input("lenght between circle and wall " ))
+import math
```
Comment (katemalash) [inline naydaN/HW4/HW4-4.py:4]: It's good practice to use import at file beginning

File: naydaN/HW4/HW4-4.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+a = float(math.sqrt(S)) 
+if int( a/2 - R >= K) :
+    print("it is possible to place stage in the hall ")
+else :
+    print("it is impossible to place stage in the hall")
```
Comment (katemalash) [inline naydaN/HW4/HW4-4.py:9]: Good Decision

File: naydaN/HW4/HW4-5.py [hunk 6]
```diff
@@ -0,0 +1,2 @@
+a = int(input("Input amount of money "))
+print( "200=", (a // 200) ,"," "100=", ((a % 200) // 100),"," "10=", ((a % 200 % 100) // 10),"," "1=", (a % 200 % 100 % 10 // 1))
```
Comment (katemalash) [inline naydaN/HW4/HW4-5.py:2]: Good Decision.

File: naydaN/HW4/HW4-6.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
+place = int(input("Input number of your sit "))
+if int(place > 0 and place < 37 and place%2 == 0) :
+ print("sit #", place, "is compartment top ")
+elif int(place > 0 and place < 37 and place%2 != 0) :
+    print("sit #", place, "is compartment bottom")
+elif int(place > 36 and place < 55 and place%2 == 0) :
+    print("sit #", place, "is not compartment top ")
+elif int(place > 36 and place < 54 and place%2 != 0) :
+     print("sit #", place, "is not compartment bottom")
```
Comment (katemalash) [inline naydaN/HW4/HW4-6.py:9]: Good Decision.

**Review by katemalash — APPROVED** [review]
In General home work is done good. You can merge pull-request after fixing some minor issues.

File: naydaN/HW4/HW4-1.py (hunk 1 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-1.py]: I inadvertently read how to count a leap year

File: naydaN/HW4/HW4-2.py (hunk 2 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-2.py]: got it!

File: naydaN/HW4/HW4-3.py (hunk 3 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-3.py:8]: )))

File: naydaN/HW4/HW4-4.py (hunk 4 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-4.py]: ok, thank's

File: naydaN/HW4/HW4-4.py (hunk 5 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-4.py:9]: ))

File: naydaN/HW4/HW4-5.py (hunk 6 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-5.py:2]: ))

File: naydaN/HW4/HW4-6.py (hunk 7 above)
Comment (NathalieNayda) [reply naydaN/HW4/HW4-6.py:9]: ))

**Pull request merged**